ros2_batch_job/linux_batch/__init__.py

```diff
@@ -53,7 +53,7 @@ def show_env(self):
         # Show the env
         self.run(['export'], shell=True)
         # Show what pip has
-        self.run(['"%s"' % self.python, '-m', 'pip', 'freeze', '--all'], shell=True)
+        self.run(['"%s"' % self.python, '-m', 'pip', 'list'], shell=True)
```

Comment:
Again, I'd actually advocate for `freeze --all` over just `list`. What prompted the change?